go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/GoogleContainerTools/kpt
 
 go 1.17
 
+replace github.com/GoogleContainerTools/kpt/porch/api => ./porch/api
```

Comment:
The current CLI seems to be incompatible with the changes I made on the backend. My temporary solution is to pin it to the local directory so that the CLI tests can pass, and then remove this replace statement immediately in a followup PR when this merges.